app/main.py

```diff
@@ -2,0 +1,41 @@
+from __future__ import annotations
+
+
+# Fixed class
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
```

Comment:
Code Style: You have used descriptive and correct variable names in your code. This makes your code more readable and maintainable. Good job!